modules/win-autoscale/watchmaker-win-autoscale.cfn.json

```diff
@@ -780,6 +836,53 @@
                                     ]
                                 }
                             },
+                            "c:\\cfn\\scripts\\AmazonCloudWatchAgent.zip": {
```

Comment:
The file retrieval mechanisms of the cfn-bootstrap utility have proven largely unreliable for our use cases. We've switched (I think everywhere) to aws-cli/powershell methods to retrieve files. This would also change the parameter to require that it begin with `s3://`. Example:

* https://github.com/plus3it/terraform-aws-watchmaker/blob/fc23b21cd9b44f2303561549f78ae92339fee588/modules/win-autoscale/watchmaker-win-autoscale.cfn.json#L605-L613